recipe/build.sh

```diff
@@ -59,5 +41,8 @@ do
 done
 
 if [[ "${CONDA_BUILD_CROSS_COMPILATION}" != "1" ]]; then
-  ctest --output-on-failure -j${CPU_COUNT}
+  # Do not run ctest on osx as it causes our ci to crash.
```

Comment:
I would make it extra clear that the OS is crashing.

Reply:
Might just be best to say "this test causes crashes on certain osx configurations".